data/helpers/actionRouter.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+
+const Action = require('../helpers/actionModel');
+
+const actionRouter = express.Router();
+
+
+// ACTION CRUD
+
+// get insert update remove
+
```

Comment:
You should have to get endpoints, one that its not expecting an id and will look for all the actions in the database, and other expecting for an id, and it will retrieve the action that matches with that id, in this endpoint you are using the Action.get method looking for an in, and you are passing it the id coming from the params, however your endpoint its not expecting any params with the name id. It should be the same set up you have on the put and delete endpoints, and also you need to implement the missing endpoint, the one that should retrieve all actions in the database